hack/tools/swagger/generate-openapi/auto_generate_openapi.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func main() {
```

Comment:
I think a shell script would be more appropriate then the gofile for execute openapi-gen command. BTW, we can use the compiled binary openapi-gen.